docs/modules/integrations/pages/zephyr-exporter.adoc

```diff
@@ -0,0 +1,160 @@
+= Zephyr Exporter
+
+Zephyr Exporter is a tool used for exporting test execution results into Jira Zephyr Plugin.
+
+Features:
+
+* Create test executions
+* Set test execution statuses
+
+== Zephyr Export Properties
+
+[cols="1,2,3", options="header"]
+|===
+
+|Property
+|Required
+|Description
+
+|`zephyr.exporter.source-directory`
+|true
+|Path to directory with test execution JSON results.
+
+|`zephyr.project-key`
+|true
+|Key of a Jira project where new test executions should be created
+
+|`zephyr.version-name`
+|true
+|Version/release name where new test executions should be created
+
+|`zephyr.cycle-name`
+|true
+|Cycle/sprint name where new test executions should be created
+
+|`zephyr.folder-name`
+|true
+|Folder name where new test executions should be created
+
+|===
+
+== Zephyr Execution Status Mapping
+
+The Zephyr plugin for Jira has own configurable execution statuses. For getting detailed information about them can be used request like https://jira.example.com/rest/zapi/latest/util/testExecutionStatus. These properties are used to establish a correspondence between Vividus and Zephyr execution statuses.
+
+[cols="1,2", options="header"]
+|===
+
+|Vividus execution status property name
+|Zephyr execution status value (by default)
+
+|`zephyr.statuses.FAILED`
+|FAILED
+
+|`zephyr.statuses.BROKEN`
+|TEST NEEDS REWORK
+
+|`zephyr.statuses.PENDING`
+|TEST NEEDS REWORK
+
+|`zephyr.statuses.UNKNOWN`
+|PASSED WITH REMARKS
+
+|`zephyr.statuses.SKIPPED`
+|SCHEDULED
+
+|`zephyr.statuses.PASSED`
+|PASSED
+
+|===
+
+== Authorization
+
+[cols="1,2", options="header"]
+|===
+
+|Name
+|Description
+
+|[#noop-link]`noop`
+|Used if Jira endpoint is hidden behind SSO and the only way to access the endpoint is to use custom authorization flows or cookies
+
+|[#caching-credentials-link]`caching-credentials`
+|Authorize in Jira using client username and password
+
+|===
+
+== Jira properties
+
+[cols="1,1,1,3", options="header"]
+|===
+
+|Property
+|Required
+|Acceptable values
+|Description
+
+|jira.username
+|Required for `caching-credentials` auth
+|`string`
+|Jira username used for exporting test execution results
+
+|jira.password
+|Required for `caching-credentials` auth
+|`string`
+|Jira password used used for exporting test execution results
+
+|jira.endpoint
+|true
+|https://en.wikipedia.org/wiki/URL[URL]
+|Jira URL e.g. `https://jira.example.com`
+
+|jira.http.context-type
+|false
+a|<<caching-credentials-link, caching-credentials>>
+
+<<noop-link, noop>>
+
+|defined Jira <<_authorization>> mechanism, default value is <<caching-credentials-link, caching-credentials>>
+
+|===
```

Comment:
the common with Xray part can be moved to partials (you can find examples in the plugins docs)